types/express-formidable/index.d.ts

```diff
@@ -1,9 +1,20 @@
 // Type definitions for express-formidable 1.0.0
 // Project: https://github.com/noraesae/express-formidable
-// Definitions by: Torkild Dyvik Olsen <https://github.com/tdolsen>
+// Definitions by: Torkild Dyvik Olsen <https://github.com/tdolsen>, Evan Shortiss <https://github.com/evanshortiss>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import { RequestHandler } from "express";
+import * as express from "express";
+import { Fields, Files } from "formidable"
+
+// Extend the express request object with attached formidable files and fields
+declare global {
+  namespace Express {
```

Comment:
Which global Express does this augment? I don't see the namespace within the `express` types.

Reply:
It's this here - https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-serve-static-core/index.d.ts#L11

Express core types come from this oddly enough.

Reply:
Weird. Okay though!